.coveragerc

```diff
@@ -177,6 +177,7 @@ omit =
     homeassistant/components/thermostat/proliphix.py
     homeassistant/components/thermostat/radiotherm.py
     homeassistant/components/*/thinkingcleaner.py
+    homeassistant/components/*/webostv.py
```

Comment:
Please add this to the top with all the other wildcard statements. Please move thinkingcleaner too